R2300:
  transport: "udp"
  scanner_ip: "10.0.10.76"
  start_angle: -450000
  max_num_points_scan: 0
  packet_type: "C1"
  scan_topic: "/cloud"
  frame_id: "scanner_link"
  watchdogtimeout: 0
  watchdog: false
  num_layers: 4
  apply_correction: true
